extensions/amp-animation/0.1/web-animations.js

```diff
@@ -761,18 +761,37 @@ export class MeasureScanner extends Scanner {
           'Both "selector" and "target" are not allowed');
       targets = this.css_.queryElements(spec.selector);
       if (targets.length == 0) {
-        user().warn(TAG, `Target not found: "${spec.selector}"`);
+        const doc = this.css_.rootNode_;
+        var allNodes = doc.getElementsByTagName('*');
+        for (var i = 0; i < allNodes.length; i++) {
+          if(allNodes[i].shadowRoot) {
```

Comment:
Why is an animation crossing a shadow? Shouldn't these be defined inside it?

Reply:
The animation is defined in the shadow. But the document being checked `this.css_.rootNode_` is the main document which like in `action_impl.js` is unable to find the target node.